File: src/B_1297/B_1297.java
package B_1297;

import java.io.*;
import java.util.*;

public class B_1297 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int c = Integer.parseInt(st.nextToken());
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());
        double res = Math.pow(c,2)/(Math.pow(a,2)+Math.pow(b,2));
        System.out.println((int)Math.sqrt(res*Math.pow(a,2))+" "+(int) Math.sqrt(res*Math.pow(b,2)));

    }
}

File: src/B_1297/B_1297.kt
package B_1297

import java.io.*
import java.util.*
import kotlin.math.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val st = StringTokenizer(readLine())
    val c = st.nextToken().toDouble()
    val a = st.nextToken().toDouble()
    val b = st.nextToken().toDouble()
    val res = c.pow(2.0) / (a.pow(2.0) + b.pow(2.0))
    print("${sqrt(res*a.pow(2)).toInt()} ${sqrt(res*b.pow(2)).toInt()}")
}